xchange-bittrex/src/main/java/org/knowm/xchange/bittrex/BittrexAdapters.java

```diff
@@ -268,9 +268,9 @@ public static void adaptMetaData(List<BittrexSymbol> rawSymbols,
 
     for (BittrexCurrency bittrexCurrency : bittrexCurrencies) {
       WalletHealth walletHealth = WalletHealth.UNKNOWN;
-      if( BITTREX_CURRENCY_ONLINE_STRING.equals(bittrexCurrency.getStatus()) ){
+      if( ONLINE.equals(bittrexCurrency.getStatus()) ){
         walletHealth = WalletHealth.ONLINE;
-      }else if( BITTREX_CURRENCY_OFFLINE_STRING.equals(bittrexCurrency.getStatus())) {
+      }else if( OFFLINE.equals(bittrexCurrency.getStatus())) {
```

Comment:
this spacing is not correct, space after if and before bracket

Reply:
no space after paren

Reply:
https://github.com/knowm/XChange/wiki/Code-Style

Reply:
Fixed in the latest commit